### PR TITLE
skip_consent_pdf_checking_for_self_request

### PR DESCRIPTION
**Pull request by wangy70**

### CODE REVIEW

File: rdr_service/app_util.py [hunk 1]
```diff
@@ -182,6 +182,10 @@ def get_account_origin_id():
     return client_id
 
 
+def is_self_request():
+    return _is_self_request()
+
+
```
Comment (kenny-skaggs) [inline rdr_service/app_util.py]: Could it make more sense to rename the `_is_self_request` method to `is_self_request`?

File: .idea/runConfigurations/Unittests__Full.xml [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
+    <option name="IS_MODULE_SDK" value="true" />
     <option name="ADD_CONTENT_ROOTS" value="true" />
     <option name="ADD_SOURCE_ROOTS" value="true" />
     <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
     <option name="SCRIPT_NAME" value="unittest" />
-    <option name="PARAMETERS" value="discover -v -s tests" />
+    <option name="PARAMETERS" value="discover -v -s tests -p &quot;*test_questionnaire_response_dao*&quot;" />
     <option name="SHOW_COMMAND_LINE" value="false" />
     <option name="EMULATE_TERMINAL" value="false" />
     <option name="MODULE_MODE" value="true" />
```
Comment (kenny-skaggs) [inline .idea/runConfigurations/Unittests__Full.xml]: Looks like changes to the full unit test configuration got uploaded.

File: rdr_service/app_util.py (hunk 1 above)
Comment (wangy70) [reply rdr_service/app_util.py]: done

File: .idea/runConfigurations/Unittests__Full.xml (hunk 2 above)
Comment (wangy70) [reply .idea/runConfigurations/Unittests__Full.xml]: Ah, it's submitted by accident...

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**